CodingChallenges/CC_117_SevenSegmentDisplay_p5.js/sketch.js

```diff
@@ -0,0 +1,50 @@
+let nums = [0x7e, 0x30, 0x6d, 0x79, 0x33, 0x5b, 0x5f, 0x70, 0x7f, 0x7b];
```

Comment:
Can you add the header information with link to the video here? Then I'll merge. Yay!!

Reply:
```suggestion
// Daniel Shiffman
// http://youtube.com/thecodingtrain
// http://codingtra.in

// Coding Challenge 118: Seven Segment Display
// YouTube URL

// p5.js editor version:
// p5.js editor url```

Reply:
Whoa I've never tried this suggestion thing before, feel free to ignore it, I didn't mean for it to remove the line of code!

Reply:
Done! 😃 Added this now (the old fashioned way), but this "suggest" thing also looks great!
Please let me know if it's still not quite right.